src/main/java/TaskTwo.java

```diff
@@ -23,4 +25,16 @@ public static String getExpandString(String[] array) {
     public static String[] splitString(String inputString) {
         return inputString.split("");
     }
+
+    public static String inputString() {
+        System.out.println("Hi! Enter any sentence and I will unfold it! But not only numbers ;)");
+        String str = new Scanner(System.in).nextLine();
```

Comment:
Please, replace "new Scanner(System.in).nextLine()" with util method from "InputReader" class